azure-pipelines.yml

```diff
@@ -32,28 +36,57 @@ resources:
       name: nanoframework/nf-tools
       endpoint: nanoframework
 
-pool:
-  vmImage: 'windows-latest'
-
-variables:
-  DOTNET_NOLOGO: true
-  solution: 'nanoFramework.Device.Can.sln'
-  buildPlatform: 'Any CPU'
-  buildConfiguration: 'Release'
-  nugetPackageName: 'nanoFramework.Device.Can'
-
-steps:
-
-# step from template @ nf-tools repo
-# all build, update and publish steps
-- template: azure-pipelines-templates/class-lib-build.yml@templates
-  parameters:
-    sonarCloudProject: 'nanoframework_lib-nanoFramework.Devices.Can'
-
-# step from template @ nf-tools repo
-# report error
-- template: azure-pipelines-templates/discord-webhook-task.yml@templates  
-  parameters:
-    status: 'failure'
-    webhookUrl: '$(DiscordWebhook)'
-    message: ''
+jobs:
```

Comment:
Why are you using 2 jobs instead of 2 tasks?
This takes so much time as it has to fire a second VM for doing the same as it could do in the same run...

Reply:
Copied from a recently updated azure pipeline yaml..  I guess we need to do a check on all of them!